src/verifast.ml

```diff
@@ -1999,7 +1999,7 @@ module VerifyProgram(VerifyProgramArgs: VERIFY_PROGRAM_ARGS) = struct
           end
           lems
       in
-      let predinstmap = localpredinsts @ predinstmap' @ predinstmap in
+      let predinstmap = predinstmap @ localpredinsts @ predinstmap' in

```

Comment:
Maak deze spurious edit ongedaan. (Voor performance moet de langste lijst in een append achteraan.) Maak trouwens alle spurious edits ongedaan.